xml/System.Linq/ParallelEnumerable.xml

```diff
@@ -3683,7 +3687,7 @@
         <remarks>To be added.</remarks>
         <exception cref="T:System.OperationCanceledException">The query was canceled with the token passed in through <paramref name="WithCancellation" />.</exception>
         <exception cref="T:System.ArgumentNullException">
-          <paramref name="source" /> or <paramref name="action" /> is a null reference (Nothing in Visual Basic).</exception>
+          <paramref name="source" /> or <paramref name="action" /> or <paramref name="keySelector" /> or <paramref name="elementSelector" /> or <paramref name="resultSelector" /> is <see langword="null" /> (Nothing in Visual Basic).</exception>
```

Comment:
```suggestion
          <paramref name="source" /> or <paramref name="action" /> or <paramref name="keySelector" /> or <paramref name="elementSelector" /> or <paramref name="resultSelector" /> is <see langword="null" /> (<see langword="Nothing" /> in Visual Basic).</exception>
```

Reply:
Good catch. I think I can ask the tool to substitute it automatically (if the whole string is `Nothing in Visual Basic`).